contribs/gmf/apps/mobile/index.html

```diff
@@ -101,7 +101,7 @@
     <script src="../../../../node_modules/bootstrap/dist/js/bootstrap.js" type="text/javascript"></script>
     <script src="../../../../node_modules/d3/d3.min.js" type="text/javascript"></script>
     <script src="../../../../node_modules/less/dist/less.min.js"></script>
-    <script src="/@?main=mobile/js/mobile.js"></script>
+    <script src="/@?main=js/mobile.js"></script>
```

Comment:
Yes i haven't tested yet the hosted example on it so it was just for testing.
I'll make the proper fix when i'm sure everything is working fine :)
Thanks